PocketBeagle/Grove/ADC.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2020 SeeedStudio
```

Comment:
Why is this re-written and why is it not in the .work-in-progress folder for now (once we complete it)?